Sources/Extensions/Foundation/String+Povio.swift

```diff
@@ -37,4 +37,14 @@ public extension String {
     let emailTest = NSPredicate(format: "SELF MATCHES %@", emailRegEx)
     return emailTest.evaluate(with: self)
   }
+  
+  /// Returns first n characters from string
+  func first(n characters: Int) -> String {
+    .init(prefix(min(count, characters)))
+  }
+  
+  /// Returns last n characters from string
+  func last(n characters: Int) -> String {
+    .init(suffix(min(count, characters)))
+  }
```

Comment:
These seem redundant, `prefix` and `suffix` already do their job, do they not?

Reply:
Yes, this is from pre-swift4 times.